models/event.js

```diff
@@ -32,6 +33,9 @@ const MODEL = {
         .string().hex().length(40)
         .description('SHA this project was built on')
         .example('ccc49349d3cffbd12ea9e3d41521480b4aa5de5f'),
+    startFrom: requiresValue
+        .description('Job name the event starts from or ~commit/~pr')
```

Comment:
Can you say something on the line `Event start point - a job name or trigger name (~commit/~pr)`

Reply:
addressed :)